src/html/logo.html

```diff
@@ -1,51 +1,58 @@
 <!-- section -->
 <section id="logo" class="section">
-  <h1 class="h1">Logo</h1>
-  <h2 class="h2">Icon</h2>
+  <h1 class="h1 weight-strong">Logo</h1>
+  <h2 class="h2 weight-strong">Icon</h2>
   <div class="figure-wrapper">
-    <figure class="figure figure-logo-icon">
-      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="190px" alt="Lockup Purple">
-      <figcaption class="figcaption">Pink Icon (RGB)</figcaption>
-      <a class="btn btn-xs white" href="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.eps" download>Download EPS</a>
-      <a class="btn btn-xs white" href="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.png" download>Download PNG</a>
-      <a class="btn btn-xs white" href="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" download>Download SVG</a>
+    <figure class="grid-block figure figure-logo-icon">
+      <div class="grid-block vertical">
```

Comment:
You don't need these divs that wrap your figure content. Instead you can remove them ad put the classes on the figure itself.